lightning/src/offers/invoice_request.rs

```diff
@@ -1121,7 +1134,7 @@ mod tests {
 
 		let invoice_request = OfferBuilder::new("foo".into(), recipient_pubkey())
 			.amount_msats(1000)
-			.supported_quantity(Quantity::one())
+			.supported_quantity(Quantity::One)
 			.build().unwrap()
 			.request_invoice(vec![1; 32], payer_pubkey()).unwrap()
 			.amount_msats(2_000).unwrap()
```

Comment:
~10 lines down, it seems like a better error now would be `InvalidQuantity`

Reply:
The quantity is still expected to not be set there, even if it were 1.

Reply:
I'm confused, I see this in the spec:

```
   - if `offer_quantity_max` is present:
      - MUST set `invreq_quantity` to greater than zero.
```
Why is the quantity not expected in the invoice request?

Reply:
`Quantity::One` is `None` for `offer_quantity_max`. We don't use an `Option` for `OfferContents::supported_quantity`.

Reply:
Oh I see. May want to clarify the difference between `Quantity::Bounded(one)` vs `Quantity::One` in the docs